src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -4536,6 +4536,9 @@ def test_ps_s3_notification_push_kafka_security_sasl_scram():
 @attr('http_test')
 def test_persistent_ps_s3_reload():
     """ do a realm reload while we send notifications """
```

Comment:
this test is supposed to reload the realm while we send notifications,
however I see we only enable v2 notifications without disabling them at the start of the test
https://github.com/ceph/ceph/pull/57639/files#diff-83b1d2148e5a0483255b9af5f707fec576625a16c04aca191578d555ca336443R4617
but we can fix it in a different PR

Reply:
can you please open a tracker (we should backport to squid)  for this?